.eslintrc.cjs

```diff
@@ -0,0 +1,114 @@
+const nodeFiles = [
+  '**/build.dflt.{js,mjs,cjs}',
+  '**/build.{js,mjs,cjs}',
+  '**/config/**',
+  '**/prebuild.{js,mjs,cjs}',
+  '**/prebuild/**',
+  '**/scripts/**',
+  'packages/new-design/lib/**',
+  'sites/backend/**',
+  'sites/*/mdx/**',
+  'sites/*/themes/**',
+]
+const frontendFiles = [
+  '**/components/**',
+  '**/hooks/**',
+  '**/pages/**',
+  '**/page-templates/**',
+  'packages/i18n/**/*.md/*.js',
+]
+const isomorphicFiles = ['packages/core/**', 'plugins/**', 'tests/designs/**', 'tests/plugins/**']
```

Comment:
Based on the [distinction](https://freesewing.dev/guides/plugins/types-of-plugins) between build-time and run-time plugins, I gathered that plugin code should be able to run in either Node.js or a web browser, and therefore ought to restrict itself to the intersection of Node builtins and browser builtins. (And I'm guessing the same is true for some of the core code.) Ironically, however, the only plugin that [seems to use](https://github.com/freesewing/freesewing/blob/a44c23c270546a2229da2a583ea33405e7bcaaab/plugins/plugin-timing/src/index.mjs#L4-L11) any such builtins actually uses items from the _union_ of Node builtins and browser builtins, and will therefore need an `eslint-ignore` directive in order to comply with the lint rules. So maybe I'm over-engineering here.